daemon/algod/api/server/v1/handlers/handlers.go

```diff
@@ -682,12 +738,17 @@ func PendingTransactionInformation(ctx lib.ReqContext, w http.ResponseWriter, r
 	}
 
 	if txn, ok := ctx.Node.GetPendingTransaction(txID); ok {
+		ledger := ctx.Node.Ledger()
 		responseTxs, err := txWithStatusEncode(txn)
 		if err != nil {
 			lib.ErrorResponse(w, http.StatusInternalServerError, err, errFailedToParseTransaction, ctx.Log)
 			return
 		}
 
+		responseTxs.TransactionResults = &v1.TransactionResults{
+			CreatedAssetIndex: computeAssetIndexFromTxn(txn, ledger),
```

Comment:
can you just add the comment that it will be omitted if there is no assetID?

Reply:
done :+1: